src/libraries/BitMath.sol

```diff
@@ -13,7 +13,7 @@ library BitMath {
         require(x > 0);
 
         assembly ("memory-safe") {
-            r := or(shl(8, iszero(x)), shl(7, lt(0xffffffffffffffffffffffffffffffff, x)))
+            r := shl(7, lt(0xffffffffffffffffffffffffffffffff, x))
             r := or(r, shl(6, lt(0xffffffffffffffff, shr(r, x))))
             r := or(r, shl(5, lt(0xffffffff, shr(r, x))))
             r := or(r, shl(4, lt(0xffff, shr(r, x))))
```

Comment:
additionally the constant on line `23` we're looking to change to
`0x0706060506020500060203020504000106050205030304010505030400000000`

given youre already doing all our changes on this file, i'd be interested in you doing it here?